mkdocs.yml

```diff
@@ -407,6 +407,7 @@ pages:
         - Caching: platform/tutorial/runtime/caching.md
         - Environment Variables: platform/tutorial/runtime/environment-variables.md
         - AMI Overview: platform/tutorial/runtime/ami-overview.md
+        - SMI aarch64 v5.9.4: platform/tutorial/runtime/aarch64-v594.md
```

Comment:
lets not add this. we're not launching an AMI here. so we dont need a page for it. since only custom nodes are supported,it'll be sufficient to update custom node docs (which have been updated) and to update the docs for drydock images.